cumulus_library/studies/discovery/code_detection.py

```diff
@@ -11,27 +12,13 @@ class CodeDetectionBuilder(base_table_builder.BaseTableBuilder):
     def _check_coding_against_db(self, code_source, schema, cursor):
         """selects the appropriate DB query to run"""
 
-        if code_source["is_array"]:
-            return sql_utils.is_field_populated(
-                schema=schema,
-                source_table=code_source["table_name"],
-                hierarchy=[(code_source["column_name"], list)],
-                expected=sql_utils.CODEABLE_CONCEPT,
-                cursor=cursor,
-            )
-        elif code_source["is_bare_coding"]:
-            return sql_utils.is_field_populated(
-                schema=schema,
-                source_table=code_source["table_name"],
-                hierarchy=[(code_source["column_name"], dict)],
-                expected=sql_utils.CODING,
-                cursor=cursor,
-            )
         return sql_utils.is_field_populated(
             schema=schema,
             source_table=code_source["table_name"],
-            hierarchy=[(code_source["column_name"], dict)],
-            expected=sql_utils.CODEABLE_CONCEPT,
+            hierarchy=code_source["column_hierarchy"],
+            expected=code_source["encoding"]
```

Comment:
Yes - but this is kind of vestigial now since i cut a lot of the rest of this out, I don't think it's actually being used anyplace, so I may just remove this check for now.